plexupdate.sh

```diff
@@ -275,7 +275,7 @@ fi
 
 # Remove any ~ or other oddness in the path we're given
 DOWNLOADDIR="$(cd ${DOWNLOADDIR// /\\ } && pwd)"
-if [ -z "${DOWNLOADDIR}" ]; then
+if [ ! -d "${DOWNLOADDIR}" ]; then
```

Comment:
wouldn't "" point to current directory and be somewhat unexpected since that should never happen? Because if cd fails, then I think downloaddir will be blank, right?